VideoRenderer/VideoRenderer/VideoStreamViewController.swift

```diff
@@ -198,6 +202,60 @@ public final class VideoStreamViewController: UIViewController, RendererProtocol
             if currentPlayer.rate != props.rate {
                 currentPlayer.rate = props.rate
             }
+            
+            if case .supported(let active) = props.pictureInPictureMode {
+                setPictureInPictureController(toState: active)
```

Comment:
Prefer having a direct implementation here. Don't see a reason to have a separate method for this.